vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -16157,6 +16229,16 @@ unit:PERCENT-M-PER-M
   rdfs:label "PERCENT M PER M" ;
   skos:prefLabel "percent metre per metre" ;
 .
+unit:PERCENT-MI-PER-MI
+  a qudt:Unit ;
+  qudt:conversionMultiplier "0.000006215040398"^^xsd:double ;
```

Comment:
Why not 0.01?

Reply:
I'm not understanding why all these dimensionless ratios don't just have 0.01 as the conversion multiplier.

Reply:
Because the numerator is implied in percent, but it was really percent-per-MI, so it would 0.01 * 1/(CM for MI). We renamed them from PERCENT-PER-<FOO> because that would seem like it isn't a percentage at all.